packages/components/components/elvis-carousel/src/react/elvia-carousel.tsx

```diff
@@ -268,7 +270,13 @@ export const Carousel: FC<CarouselProps> = function ({
               isSelected={listIndex === index}
               tabIndex={0}
               aria-label={
-                listIndex === index ? `Du er på side ${listIndex + 1}` : `Gå til side ${listIndex + 1}`
+                listIndex === index
+                  ? lang === 'no'
+                    ? `Du er på side ${listIndex + 1}`
+                    : `You are on page ${listIndex + 1}`
+                  : lang === 'no'
+                    ? `Gå til side ${listIndex + 1}`
+                    : `Go to page ${listIndex + 1}`
```

Comment:
Consider refactoring these nested ternary statements to a function `getAriaLabel` (or something like that) to make it a bit more readable.